Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Chapel/bibles.yml

```diff
@@ -13,3 +13,22 @@
   - type: Item
     size: 15
     sprite: Nyanotrasen/Objects/Specific/Chapel/holylight.rsi
+
+- type: entity
+  parent: Bible
+  id: BibleChaplain
+  description: The chaplain's holy book. Comes with a handheld blessing dispenser.
+  components:
+  - type: Storage
+    capacity: 10
+  - type: StorageFill
+    contents:
+      - id: WeaponRevolverFaith
```

Comment:
when originally designing the chaplain for ss14 there was a discussion over what to give them (null rod, faith, something else?) and we decided on familiars filling this role

Reply:
Yes, but having it in the game would still be nice, especially given some of the potential ideas for the future of Nyano.

Reply:
why does this role need a gun...?

Reply:
It's not the role it's the gun itself being in the game. Lmao.

Reply:
For the gun to be in the game, it has to come from somewhere. There's no need to have an unused entity.

Reply:
I would argue that 80% of the guns in the game have to be mapped in anyway but I'll remove Faith and the special bible (that I originally forgot to remove anyway)